data/blueprints/README.md

```diff
@@ -1,756 +1,4 @@
-
-* As of DF 0.34.x, it is no longer possible to build doors (d) at the same time that you build adjacent walls (Cw). Doors must now be built *after* walls are constructed for them to be next to. This does not affect the more common case where walls exist as a side-effect of having dug-out a room in a #dig blueprint.
-
-
-Caveats and limitations
------------------------
-
-* Buildings will be designated regardless of whether you have the required materials, but if materials are not available when the construction job is picked up by a dwarf, the buildings will be canceled and the designations will disappear. Until the buildingplan plugin can be extended to support all building types, you should use `quickfort orders` to pre-manufacture all the materials you need for a `#build` blueprint before you apply it.
-
-* If you use the `jugs` alias in your `#query`-mode blueprints, be aware that there is no way to differentiate jugs from other types of tools in the game. Therefore, `jugs` stockpiles will also take nest boxes and other tools. The only workaround is not to have other tools lying around in your fort.
-
-* Likewise for bags. The game does not differentiate between empty and full bags, so you'll get bags of gypsum power and sand in your bags stockpile unless you avoid collecting sand and are careful to assign all your gypsum to your hospital.
-
-* Weapon traps and upright spear/spike traps can currently only be built with a single weapon.
-
-* Pressure plates can be built, but they cannot be usefully configured yet.
-
-* Building instruments, bookcases, display furniture, and offering places are not yet supported by DFHack.
-
-* This script is relatively new, and there are bound to be bugs! Please report them at the [DFHack issue tracker](https://github.com/DFHack/dfhack/issues) so they can be addressed.
-
-
-Links
------
-
-### Quickfort links ###
-
-* [Quickfort command syntax](https://docs.dfhack.org/en/stable/docs/_auto/base.html#quickfort)
-* [Quickfort forum thread](http://www.bay12forums.com/smf/index.php?topic=176889.0)
-* [Quickfort blueprints library](https://github.com/DFHack/dfhack/tree/develop/data/blueprints/library)
-* [DFHack issue tracker](https://github.com/DFHack/dfhack/issues)
-* [Quickfort source code](https://github.com/DFHack/scripts/tree/master/internal/quickfort)
-
-### Related tools ###
-
-* DFHack's [blueprint plugin](https://docs.dfhack.org/en/stable/docs/Plugins.html#blueprint) can generate blueprints from actual DF maps.
-* [Python Quickfort](http://joelpt.net/quickfort) is the previous, Python-based implementation that DFHack's quickfort script was inspired by.
+* [Quickfort overview](https://docs.dfhack.org/en/stable/docs/_auto/base.html#quickfort)
```

Comment:
suggest: Quickfort overview -> Quickfort command reference

Reply:
Sounds good, I changed it in the user guide too